caps/types.go

```diff
@@ -53,6 +54,13 @@ func (t *BoolFileType) Name() string {
 	return "bool-file"
 }
 
+var boolFileAllowedPathPatterns = []*regexp.Regexp{
+	// The brightness of standard LED class device
+	regexp.MustCompile("^/sys/class/leds/[^/]+/brightness$"),
```

Comment:
Sorry, I take that back. Yes, these cases indeed work with 0 or 1, so we can instead blacklist whatever we don't want later.